tools/cldr-code/src/main/resources/org/unicode/cldr/tool/modify_config.txt

```diff
@@ -1,151 +1,124 @@
-locale=doi ; action=add ; new_path=//ldml/personNames/nameOrderLocales[@order="givenFirst"]; new_value=und doi
-locale=el ; action=add ; new_path=//ldml/personNames/nameOrderLocales[@order="givenFirst"]; new_value=und el
-locale=ff ; action=add ; new_path=//ldml/personNames/nameOrderLocales[@order="givenFirst"]; new_value=und ff
-locale=ff_Adlm ; action=add ; new_path=//ldml/personNames/nameOrderLocales[@order="givenFirst"]; new_value=und ff
-locale=fo ; action=add ; new_path=//ldml/personNames/nameOrderLocales[@order="givenFirst"]; new_value=und fo
-locale=ha ; action=add ; new_path=//ldml/personNames/nameOrderLocales[@order="givenFirst"]; new_value=und ha
-locale=kgp ; action=add ; new_path=//ldml/personNames/nameOrderLocales[@order="givenFirst"]; new_value=und kgp
-locale=ks ; action=add ; new_path=//ldml/personNames/nameOrderLocales[@order="givenFirst"]; new_value=und ks
-locale=ks_Deva ; action=add ; new_path=//ldml/personNames/nameOrderLocales[@order="givenFirst"]; new_value=und ks
-locale=ky ; action=add ; new_path=//ldml/personNames/nameOrderLocales[@order="givenFirst"]; new_value=und ky
-locale=lo ; action=add ; new_path=//ldml/personNames/nameOrderLocales[@order="givenFirst"]; new_value=und lo
-locale=mai ; action=add ; new_path=//ldml/personNames/nameOrderLocales[@order="givenFirst"]; new_value=und mai
-locale=mni ; action=add ; new_path=//ldml/personNames/nameOrderLocales[@order="givenFirst"]; new_value=und mni
-locale=ne ; action=add ; new_path=//ldml/personNames/nameOrderLocales[@order="givenFirst"]; new_value=und ne
-locale=qu ; action=add ; new_path=//ldml/personNames/nameOrderLocales[@order="givenFirst"]; new_value=und qu
-locale=sa ; action=add ; new_path=//ldml/personNames/nameOrderLocales[@order="givenFirst"]; new_value=und sa
-locale=sat ; action=add ; new_path=//ldml/personNames/nameOrderLocales[@order="givenFirst"]; new_value=und sat
-locale=sd_Deva ; action=add ; new_path=//ldml/personNames/nameOrderLocales[@order="givenFirst"]; new_value=und sd
-locale=sr_Latn ; action=add ; new_path=//ldml/personNames/nameOrderLocales[@order="givenFirst"]; new_value=und sr
-locale=su ; action=add ; new_path=//ldml/personNames/nameOrderLocales[@order="givenFirst"]; new_value=und su
-locale=tg ; action=add ; new_path=//ldml/personNames/nameOrderLocales[@order="givenFirst"]; new_value=und tg
-locale=ti ; action=add ; new_path=//ldml/personNames/nameOrderLocales[@order="givenFirst"]; new_value=und ti
-locale=tt ; action=add ; new_path=//ldml/personNames/nameOrderLocales[@order="givenFirst"]; new_value=und tt
-locale=xh ; action=add ; new_path=//ldml/personNames/nameOrderLocales[@order="givenFirst"]; new_value=und xh
-locale=wo ; action=add ; new_path=//ldml/personNames/nameOrderLocales[@order="givenFirst"]; new_value=und yo
-locale=yrl ; action=add ; new_path=//ldml/personNames/nameOrderLocales[@order="givenFirst"]; new_value=und yrl
\ No newline at end of file
+# CLDR-14951 english date format separators for ch and ln
```

Comment:
Is it normal that the CLDRModify info is also included in the PR?

Reply:
Yes, just to leave multiple examples of how it is done.